icu4c/source/i18n/unum.cpp

```diff
@@ -28,17 +28,83 @@
 #include "unicode/dcfmtsym.h"
 #include "unicode/curramt.h"
 #include "unicode/localpointer.h"
+#include "unicode/measfmt.h"
 #include "unicode/udisplaycontext.h"
 #include "uassert.h"
 #include "cpputils.h"
 #include "cstring.h"
+#include "putilimp.h"
 
 
 U_NAMESPACE_USE
 
 
+// to support UNUM_DURATION
+class MeasureFormatWrapper : public NumberFormat {
```

Comment:
Suggestion: `duration` should be in the name somewhere, either `MeasureFormatDurationWrapper` or `MeasureFormatWrapper::formatDuration` (probably the latter)